### PR TITLE
✨ Support connect wallet, show, buy and create listings

### PR DESCRIPTION
**Pull request by williamchong**
image
image

### CODE REVIEW

File: pages/sell/[classId]/[nftId].vue [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+const route = useRoute();
+const store = useWalletStore();
+const { wallet, signer } = storeToRefs(store);
+const classData = ref({} as any);
+const nftData = ref({} as any);
+const listingPrice = ref(32);
+const listingExpiration = ref(new Date(Date.now() + 15552000000).toISOString().split('T')[0]);
+
+const maxExpirationValue = computed(() => new Date(Date.now() + 15552000000).toISOString().split('T')[0]);
+const minExpirationValue = computed(() => new Date(Date.now()).toISOString().split('T')[0]);
```
Comment (nnkken) [inline pages/sell/[classId]/[nftId].vue:57]: I think `computed` doesn't work with `Date.now()` since the function is not reactive?

Comment (williamchong) [reply pages/sell/[classId]/[nftId].vue:57]: Yes seems nothing is changing anyway, just data would do

File: pages/latest/purchase.vue [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    <table>
+      <thead>
+        <tr>
+          <th>#</th>
+          <th>NFT information</th>
+          <th>Purchase Price</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="item, i in buyNFTEvents">
```
Comment (AuroraHuang22) [inline pages/latest/purchase.vue:13]: Should provide a unique key attribute for each item
```suggestion
        <tr v-for="item, i in buyNFTEvents" :key=“item.nft_id”>
```

File: pages/listings/[classId].vue [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      <table v-else>
+        <thead>
+          <tr>
+            <th>#</th>
+            <th>NFT information</th>
+            <th>Listing Price</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr v-for="item, i in listing">
```
Comment (AuroraHuang22) [inline pages/listings/[classId].vue:24]: Should provide a unique key for each item

File: pages/owned/index.vue [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+      <table v-else>
+        <thead>
+          <tr>
+            <th>#</th>
+            <th>NFT information</th>
+            <th>Your Listing</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr v-for="item, i in ownedList">
```
Comment (AuroraHuang22) [inline pages/owned/index.vue:16]: Should provide a unique key for each item

File: pages/index.vue [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+      <table>
+        <thead>
+          <tr>
+            <th>#</th>
+            <th>NFT information</th>
+            <th>Listing Price</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr v-for="item, i in listingEvents">
```
Comment (AuroraHuang22) [inline pages/index.vue:14]: Should provide a unique key for each item

**Review by AuroraHuang22 — APPROVED** [review]

**Pull request merged**